oauth.c

```diff
@@ -934,7 +934,7 @@ static long make_req_streams(php_so_object *soo, const char *url, const smart_st
 #if OAUTH_USE_CURL
 static size_t soo_read_response(char *ptr, size_t size, size_t nmemb, void *ctx) /* {{{ */
 {
-	uint relsize;
```

Comment:
This one should be `size_t`, as it is the result of multiplying two size_t's.

Reply:
OK! I will not modify the PR anymore, because I am not that familiar with the OAuth details. I will close the PR when a fix has been applied.